spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'Get id' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to be_successful
+    end
+
+    it 'renders template index' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'renders template show' do
+      get '/users/:id'
+      expect(response).to render_template(:show)
+    end
```

Comment:
- According to the project requirements, for each action you are required to check `If the response body includes correct placeholder text`. Please check if the response for the `user_show` page renders the right placeholder